### DISCUSSION TITLE
GitHub Copilot extension for JetBrains  does not work when access is granted by an organization

### DISCUSSION THREAD

**Discussion by maxmalov** [community/community#67042] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I'm using WebStorm 2023.2.1 and Copilot plugin 1.2.22.3129. The Copilot plugin does not work.

I have access to GitHub Copilot from my GitHub organization. It provides me with suggestions in VSCode when I log in with the same GitHub account. However, it does not work in WebStorm. I've double-checked that the JetBrains IDE Integration OAuth application has access to the organization.

`idea.log` entries for Copilot extension

```log
2023-09-14 10:43:49,288 [  87999]   FINE - #com.github.copilot.editor.CopilotCommandListener - command modified document: Rollback Change
2023-09-14 10:43:49,288 [  87999]   FINE - #com.github.copilot.editor.CopilotEditorManagerImpl - editorModified
2023-09-14 10:43:49,288 [  87999]   FINE - #com.github.copilot.editor.CopilotEditorManagerImpl - Completions unavailable because user isn't authorized for GitHub Copilot
```

<img width="784" alt="Screenshot 2023-09-14 at 10 57 15 AM" src="https://github.com/community/community/assets/284129/57c1f29e-1685-4694-8edc-9a6a3b83eefc">
<img width="761" alt="Screenshot 2023-09-14 at 10 58 27 AM" src="https://github.com/community/community/assets/284129/9c4732e9-0d64-464d-8911-ea428e013f8b">

I believe the Copilot plugin for IntelliJ could use the same authorization logic as VSCode extension.

**Comment by bnorman21** [comment]
I have the same issue

**Comment by shu512** [comment]
May help someone: I had the same issue, but turning on VPN helped :)
